health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/AppConfigurationParentLayer.js

```
@@ -0,0 +1,247 @@
+  const [numberTabs, setNumberTabs] = useState([]);
+  const [currentStep, setCurrentStep] = useState(1);
+  const [stepper, setStepper] = useState([]);
+  const [showToast, setShowToast] = useState(null);
+  const [currentScreen, setCurrentScreen] = useState({});
+  console.log("PARENT DATA", parentState)
+  const { isLoading: isLoadingAppConfigMdmsData, data: AppConfigMdmsData } = Digit.Hooks.useCustomMDMS(
+    Digit.ULBService.getCurrentTenantId(),
+    MODULE_CONSTANTS,
+    [{ name: masterName }],
+    {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)